chromium_src/ui/views/controls/button/md_text_button.h

```diff
@@ -55,7 +55,7 @@ class VIEWS_EXPORT MdTextButton : public MdTextButtonBase {
   Kind GetKind() const;
   void SetKind(Kind kind);
 
-  void SetIcon(const gfx::VectorIcon* icon);
+  void SetIcon(const gfx::VectorIcon* icon, int icon_size = 0);
```

Comment:
Sorry, I couldn't find the usage of |icon_size|. Could you tell me more about this?

Reply:
https://source.chromium.org/chromium/chromium/src/+/main:ui/base/models/image_model.h;l=92;bpv=1;bpt=1?q=FromVectorIcon&ss=chromium%2Fchromium%2Fsrc

Basically, you can override the size of the icon (and if it's `0` it uses the default icon size).

Reply:
Now I can see where you're coming from! Thanks!